Xamarin.Forms.Platform.Android/Renderers/EditorRenderer.cs

```diff
@@ -89,6 +89,26 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 			base.OnElementPropertyChanged(sender, e);
 		}
 
+		protected override void Dispose(bool disposing)
+		{
+			if (_disposed)
+			{
+				return;
+			}
+
+			_disposed = true;
+
+			if (disposing)
+			{
+				if (Control != null)
+				{
+					Control.OnKeyboardBackPressed -= OnKeyboardBackPressed;
```

Comment:
It might. I've called out the potential break in the PR description; I don't think it's likely a problem (and it's an easy change to make), but again I'll give @jassmith a veto opportunity.